connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -104,6 +81,33 @@ public DelegatingClassLoader(List<Path> pluginLocations) {
         this(pluginLocations, DelegatingClassLoader.class.getClassLoader());
     }
 
+    public Set<PluginSource> sources() {
+        Set<PluginSource> pluginLoaders = new HashSet<>();
```

Comment:
Nit:
```suggestion
        Set<PluginSource> pluginSources = new HashSet<>();
```

(Or just `result` would be fine too)